soh/soh/Enhancements/controls/GameControlEditor.cpp

```diff
@@ -252,19 +251,21 @@ namespace GameControlEditor {
                                                 "gFirstPersonCameraSensitivityX", 0.01f, 5.0f, "", 1.0f, true);
             UIWidgets::EnhancementSliderFloat("Aiming/First-Person Vertical Sensitivity: %d %%", "##FirstPersonSensitivity Vertical",
                                               "gFirstPersonCameraSensitivityY", 0.01f, 5.0f, "", 1.0f, true);
+        } else {
+            CVarSetFloat("gFirstPersonCameraSensitivity", 1.0f);
         }
         UIWidgets::Spacer(0);
-        LUS::EndGroupPanel();
+        LUS::Context::GetInstance()->GetWindow()->GetGui()->EndGroupPanel(0);
 
         UIWidgets::Spacer(0);
-        LUS::BeginGroupPanel("Third-Person Camera", ImGui::GetContentRegionAvail());
+        LUS::Context::GetInstance()->GetWindow()->GetGui()->BeginGroupPanel("Third-Person Camera", ImGui::GetContentRegionAvail());
 
         UIWidgets::PaddedEnhancementCheckbox("Free Camera", "gFreeCamera");
         DrawHelpIcon("Enables free camera control\nNote: You must remap C buttons off of the right stick in the "
                             "controller config menu, and map the camera stick to the right stick.");
         UIWidgets::PaddedEnhancementCheckbox("Invert Camera X Axis", "gInvertXAxis");
         DrawHelpIcon("Inverts the Camera X Axis in:\n-Free camera");
-        UIWidgets::PaddedEnhancementCheckbox("Invert Camera Y Axis", "gInvertYAxis", true, true, false, "", UIWidgets::CheckboxGraphics::Cross, true);
+        UIWidgets::PaddedEnhancementCheckbox("Invert Camera Y Axis", "gInvertYAxis");
```

Comment:
why was the default value here lost?

Reply:
branch started before that change (may 7th vs may 10th), i'm waiting on a bunch of PRs to get merged then i'll merge dev into this